lib/SPIRV/SPIRVLowerBitCastToNonStandardType.cpp

```diff
@@ -77,7 +77,7 @@ static Value *removeBitCasts(Value *OldValue, Type *NewTy, NFIRBuilder &Builder,
   if (auto *LI = dyn_cast<LoadInst>(OldValue)) {
     Builder.SetInsertPoint(LI);
     Value *Pointer = LI->getPointerOperand();
-    if (!Pointer->getType()->isOpaquePointerTy()) {
+    if (!Pointer->getType()->isPointerTy()) {
```

Comment:
`!Pointer->getType()->isPointerTy()` is always false, as Pointer is obtained from Load->getPointerOperand